src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -701,18 +701,48 @@ private static int calculateTagColumn(Matcher javadocTagMatcher,
             final List<DetailAST> throwLiterals = findTokensInAstByType(blockAst,
                     TokenTypes.LITERAL_THROW);
             for (DetailAST throwAst : throwLiterals) {
-                final DetailAST newAst = throwAst.getFirstChild().getFirstChild();
-                if (newAst.getType() == TokenTypes.LITERAL_NEW) {
-                    final FullIdent ident = FullIdent.createFullIdent(newAst.getFirstChild());
-                    final ExceptionInfo exceptionInfo = new ExceptionInfo(
-                            createClassInfo(new Token(ident), currentClassName));
-                    returnValue.add(exceptionInfo);
+                if (!isInTryOrLambda(blockAst, throwAst)) {
+                    final DetailAST newAst = throwAst.getFirstChild().getFirstChild();
+                    if (newAst.getType() == TokenTypes.LITERAL_NEW) {
+                        final FullIdent ident = FullIdent.createFullIdent(newAst.getFirstChild());
+                        final ExceptionInfo exceptionInfo = new ExceptionInfo(
+                                createClassInfo(new Token(ident), currentClassName));
+                        returnValue.add(exceptionInfo);
+                    }
                 }
             }
         }
         return returnValue;
     }
 
+    /**
+     * Checks if a 'throw' usage is contained within a try block or lambda expression.
+     * Note that a try block without catch is not considered.
+     * @param methodBodyAst DetailAST node representing the method body
+     * @param throwAst DetailAST node representing the 'throw' literal
+     * @return true if throwAst is inside a try block or lambda
+     */
+    private static boolean isInTryOrLambda(DetailAST methodBodyAst, DetailAST throwAst) {
```

Comment:
Please rename method to state that we check only try with catch.
`isInTryWithCatchOrLambda` or `isInTryCatchOrLambda` or something else

Reply:
Changed to `isInTryCatchOrLambda`